### PR TITLE
fix error message for pdb.go

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
fix error message for pdb.go

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/pdb.go [hunk 1]
```diff
@@ -214,7 +214,7 @@ func (s *PodDisruptionBudgetV2Generator) validate() error {
 		return fmt.Errorf("a selector must be specified")
 	}
 	if len(s.MaxUnavailable) > 0 && len(s.MinAvailable) > 0 {
-		return fmt.Errorf("exactly one of min-available and max-available must be specified")
+		return fmt.Errorf("minAvailable and maxUnavailable cannot be both specified")
```
Comment (xiangpengzhao) [inline pkg/kubectl/pdb.go]: It'd be better to use the param names `min-available` and `max-unavailable` here instead of var names in code.

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

File: pkg/kubectl/pdb.go (hunk 1 above)
Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/pdb.go]: thanks , you are right.=)

**Comment by zhangxiaoyu-zidif** [comment]
@smarterclayton
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
ping @deads2k 
PTAL

File: pkg/kubectl/pdb.go [hunk 2]
```diff
@@ -214,7 +214,7 @@ func (s *PodDisruptionBudgetV2Generator) validate() error {
 		return fmt.Errorf("a selector must be specified")
 	}
 	if len(s.MaxUnavailable) > 0 && len(s.MinAvailable) > 0 {
-		return fmt.Errorf("exactly one of min-available and max-available must be specified")
+		return fmt.Errorf("min-available and max-unavailable cannot be both specified")
```
Comment (deads2k) [inline pkg/kubectl/pdb.go]: @davidopp I'm not sure if the code is correct or if the string is correct.

Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/pdb.go]: /assign davidopp

Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/pdb.go]: @deads2k 
image
from : https://kubernetes.io/docs/tasks/run-application/configure-pdb/
I think it is just string error. =)

**Comment by zhangxiaoyu-zidif** [comment]
ping @davidopp  @deads2k 
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

**Pull request merged**